internal/compose/compose.go

```diff
@@ -275,54 +275,58 @@ func (p *Project) WaitForHealthy(opts CommandOptions) error {
 		return err
 	}
 
+	timeout := time.After(10 * time.Minute)
```

Comment:
I was going to suggest to use `context.WithTimeout()`, so we can also interrupt calls to docker if they hang, but leveraging this context would require more changes in `internal/docker`.